app/views/search/search_boxed.html.erb

```diff
@@ -7,11 +7,14 @@
         articles and journals
       <% end %>
 
-      for 
+      for
 
       <span class="search-string">"<%= params[:q] %>"</span>
     </h2>
     <p class="subline">
+        <%= page_entries_info @pageable_results, entry_name: 'result' %>, sorted by relevance
```

Comment:
This is super minor as it falls into the "I don't care much about the html templates as they are always a mess anyway" rule... but Rails convention is two space indents not this god awful python four space indent ;)